packages/app/src/lib/import.ts

```diff
@@ -18,7 +18,7 @@ export interface ImportCSVColumn {
     name: string;
     displayName: string;
     type: FieldType | "name" | "tags";
-    exampleValues: string;
+    values: string[];
```

Comment:
Formatting the example values for display is a concern of the UI, so I thought is was better to push the "stringifying" further up the chain.